include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -615,12 +549,14 @@ struct inplace_vector
   }
 
 private: // Utilities
-  constexpr void assert_iterator_in_range(const_iterator it) noexcept {
+  constexpr void
+  assert_iterator_in_range([[maybe_unused]] const_iterator it) noexcept {
```

Comment:
These functions are left over artifacts from the original reference implementation. iirc they're not used anywhere, `IV_EXPECT` is defined as a no-op currently, I think it's safe to remove all these artifacts